pyproject.toml

```diff
@@ -30,13 +30,14 @@ dependencies = [
     "langchain-aws ~=0.2.14",
     "langchain-ollama ~=0.2.3",
     "langgraph ~=0.3.5",
+    "langchain-chroma ~=0.2.2",
     "langgraph-checkpoint-sqlite ~=2.0.1",
     "langgraph-checkpoint-postgres ~=2.0.13",
     "langgraph-supervisor ~=0.0.8",
     "langsmith ~=0.1.145",
     "numexpr ~=2.10.1",
-    "numpy ~=1.26.4; python_version <= '3.12'",
-    "numpy ~=2.2.3; python_version >= '3.13'",
+    "numpy ~=1.26.2; python_version <= '3.12'",
+    "numpy ~=1.26.2; python_version >= '3.13'",
```

Comment:
This conflicts with some later changes, I was trying locally to rebase and get uv lock to pass and having a very bad time.

It seems like LangChain team recently patched langchain-chroma to support later numpy versions (https://github.com/langchain-ai/langchain/pull/30714) but they haven't released it yet. I tried installing from git but uv was giving me a bad time.

Seems like we need to either wait for langchain-chroma 0.2.3 to be released with the latest update (preferred) or otherwise install from source to fix the conflict.